Dashboards/dashboard-Incidents.json

```diff
@@ -0,0 +1,491 @@
+						"fromValue": 7,
+						"field": ""
+					},
+					"fromDateLicense": "0001-01-01T00:00:00Z"
+				},
+				"params": {
+					"groupBy": [
+						"occurred(d)",
+						"type"
+					]
+				}
+			}
+		},
+		{
+			"id": "90f554c0-e4ff-11e7-b60b-df7995291baf",
+			"forceRange": false,
+			"x": 0,
+			"y": 4,
+			"i": "90f554c0-e4ff-11e7-b60b-df7995291baf",
+			"w": 6,
+			"h": 2,
+			"widget": {
+				"id": "acd61a60-26f0-475f-8702-f36ccb16b08e",
+				"version": 1,
+				"modified": "2017-12-19T22:59:39.708689502+02:00",
+				"name": "Incident Severity by Type",
+				"dataType": "incidents",
+				"widgetType": "bar",
+				"query": "-category:job and -status:archived and -status:closed",
+				"isPredefined": true,
+				"dateRange": {
+					"fromDate": "0001-01-01T00:00:00Z",
+					"toDate": "0001-01-01T00:00:00Z",
+					"period": {
+						"byTo": "",
+						"byFrom": "days",
+						"toValue": null,
+						"fromValue": 7,
+						"field": ""
+					},
+					"fromDateLicense": "0001-01-01T00:00:00Z"
+				},
+				"params": {
+					"groupBy": [
+						"severity",
+						"type"
+					]
+				}
+			}
+		},
+		{
+			"id": "91dc8070-e4ff-11e7-b60b-df7995291baf",
+			"forceRange": false,
+			"x": 0,
+			"y": 2,
+			"i": "91dc8070-e4ff-11e7-b60b-df7995291baf",
+			"w": 3,
+			"h": 2,
+			"widget": {
+				"id": "2945de28-7c35-42f0-a87b-8e17eadd82da",
+				"version": 1,
+				"modified": "2017-12-19T22:59:39.709196548+02:00",
+				"name": "Incident Types by Severity",
+				"dataType": "incidents",
+				"widgetType": "bar",
+				"query": "-category:job and -status:archived and -status:closed",
+				"isPredefined": true,
+				"dateRange": {
+					"fromDate": "0001-01-01T00:00:00Z",
+					"toDate": "0001-01-01T00:00:00Z",
+					"period": {
+						"byTo": "",
+						"byFrom": "days",
+						"toValue": null,
+						"fromValue": 7,
+						"field": ""
+					},
+					"fromDateLicense": "0001-01-01T00:00:00Z"
+				},
+				"params": {
+					"groupBy": [
+						"type",
+						"severity"
+					]
+				}
+			}
+		},
+		{
+			"id": "966d2c20-e4ff-11e7-b60b-df7995291baf",
+			"forceRange": false,
+			"x": 3,
+			"y": 0,
+			"i": "966d2c20-e4ff-11e7-b60b-df7995291baf",
+			"w": 3,
+			"h": 1,
+			"widget": {
+				"id": "",
+				"version": 0,
+				"modified": "2017-12-19T22:59:39.713469076+02:00",
+				"name": "Unassigned Incidents",
+				"dataType": "incidents",
+				"widgetType": "trend",
+				"query": "-category:job and -status:archived and -status:closed and owner:\"\"",
+				"isPredefined": false,
+				"dateRange": {
+					"fromDate": "0001-01-01T00:00:00Z",
+					"toDate": "0001-01-01T00:00:00Z",
+					"period": {
+						"byTo": "",
+						"byFrom": "days",
+						"toValue": null,
+						"fromValue": 7,
+						"field": ""
+					},
+					"fromDateLicense": "0001-01-01T00:00:00Z"
+				},
+				"params": {
+					"colors": {
+						"isEnabled": true,
+						"items": {
+							"#00CD33": {
+								"value": 0
+							},
+							"#FAC100": {
+								"value": 0
```

Comment:
Orange should be 1, (already got 0 as green).

Reply:
@orius123 no need to fix this?

Reply:
Nope